galsim/wfirst/__init__.py

```diff
@@ -231,7 +232,7 @@
                         [0.001270391, 0.016129619, 0.001200137] ])
 ipc_kernel /= np.sum(ipc_kernel)
 ipc_kernel = galsim.Image(ipc_kernel)
-persistence_coefficients = np.array([0.2246,0.0225,0.0085,0.0043,0.0025,0.0016,0.0011,0.0008])/100.
+persistence_coefficients = np.array([0.045707683,0.014959818,0.009115737,0.00656769,0.005135571,0.004217028,0.003577534,0.003106601])/100.
```

Comment:
We don't have any tests of most of these numbers.  Not exactly sure what those tests should be, but maybe worth adding some dummy tests in the test suite to remind ourselves of what we're missing.  E.g.
```
def test_wfirst_persistence():
    """Test the WFIRST persistence coefficients against official WFIRST project data
    """
    # Values from WFIRST project office.
    ref_coeffs =[0.045707683, 0.014959818, 0.009115737, 0.00656769, 0.005135571, 0.004217028, 0.003577534, 0.003106601] # percent
    ref_coeffs = np.array(ref_coeffs) / 100.
    np.testing.assert_array_equal(galsim.wfirst.persistence_coefficients, ref_coeffs)
    # TODO: Some kind of better check here?  Or maybe reference against an official WFIRST document/white paper, etc.?
```
Maybe this is gratuitous, but I'm always a little worried when code changes and no test needed to change along with it.

Reply:
OK... that's pretty gratuitous, and I was going to save beefing up the unit tests for the open issue on that topic, but I guess it's better to get this in before we tag v1.5.  Done.